ansible_3par_docker_plugin/container/README.md

```diff
@@ -0,0 +1,14 @@
+# Docker Volume Plugin Installer - Docker Image
+
+This is a Alpine based image with Ansible and its dependency installed along with the latest HPE 3PAR docker volume plugin ansible installer playbooks.
+
+Usage:
+
+- Run the latest docker image from docker hub, the command below will run the pre built container and open shell
+  - `docker run -it sgkasif/hpe3par_docker_volume_plugin_installer /bin/sh`
```

Comment:
This will need to change to hpestorage/... after the container is pushed to that docker hub repo